src/main/resources/io/jenkins/plugins/pipelinegraphview/consoleview/PipelineConsoleViewAction/index.jelly

```diff
@@ -8,6 +8,18 @@
             <j:out value="${h.generateConsoleAnnotationScriptAndStylesheet()}"/>
 
             <j:set var="controls">
+                <j:if test="${it.building}">
+                    <l:hasPermission permission="${it.cancelPermission}">
+                        <j:set var="proxyId" value="${h.generateId()}" />
+                        <st:bind value="${it}" var="cancelAction${proxyId}"/>
+                        <button id="pgv-cancel" data-success-message="${%Build cancelled}"
+                                data-proxy-name="cancelAction${proxyId}"
+                                class="jenkins-button jenkins-!-error-color">
+                            <l:icon src="symbol-stop-circle-outline plugin-ionicons-api"/>
+                            ${%Cancel}
```

Comment:
In my opinion you should not invent a new terminology. The existing one in Jenkins is _Abort_.

In the future this button could change dynamically from _Abort_ to _Kill_ when a aborted build is stuck (this is an existing feature of Jenkins and a special log is printed whenever that happens, meaning the plugin should be able to detect that situation).

Reply:
I'm not inventing new terminology. `Cancel` is used in the tooltip of the cancel/abort button on the main page:
<img width="263" alt="image" src="https://github.com/user-attachments/assets/41ae2079-518b-48cc-9a5a-ca7bcbf9b429" />

Reply:
Fair enough. My bad, sorry.

Reply:
Maybe you should retain the cross icon though?

Reply:
I was thinking of saving the cross x symbol for the terminate/kill versions of the button. I chose the stop symbol based on what blue ocean used.

Reply:
Interesting. I think we can afford some design freedom indeed.

I, personally, would still favor the cross icon for consistency, but that doesn't mean you should change it.

I hope there will be a day where Blue Ocean is no longer the page where people spend most of their time on, and therefore we don't have to carry on the design decisions of it.

Reply:
I'm open to changing it. Let's see if the maintainers have an opinion.

Reply:
How about calling it _Stop_?

It would allow a different icon without using the same name.

At least the API is called `doStop()` and the REST API is `/stop`. And Blue Ocean also calls it _Stop_.

And the icon you chose is a precise representation of _Stop_ (not _Cancel_).

Reply:
I'm happy as-is but will see if Jan has an opinion

Reply:
I've checked with Jan, Cancel is his preference